native_client/deepspeech.cc

```diff
@@ -112,6 +114,7 @@ struct StreamingState {
   vector<float> mfcc_buffer;
   vector<float> batch_buffer;
   ModelState* model;
+  bool extended_output; // Flag to show timings
```

Comment:
I really think that the only information stored here should be not a boolean, but the actual timing informations.